stdlib/REPL/src/docview.jl

```diff
@@ -645,7 +645,17 @@ accessible(mod::Module) =
            map(names, moduleusings(mod))...;
            collect(keys(Base.Docs.keywords))] |> unique |> filtervalid
 
-doc_completions(name) = fuzzysort(name, accessible(Main))
+function doc_completions(name) 
+    res = fuzzysort(name, accessible(Main))
+
+    # to insert an entry like `raw""` for `"@raw_str"` in `res`
+    ms = match.(r"^@(.*?)_str$", res)
+    idxs = findall(!isnothing, ms)
+    for i in idxs
+        insert!(res, i, "$(only(ms[i].captures))\"\"")
+    end
```

Comment:
After your first insertion, I think all of the subsequent indexes will be off-by-one, and then after the next, off-by-two, etc.

You can avoid this by inserting in reverse order:
```suggestion
    for i in reverse(idxs)
        insert!(res, i, "$(only(ms[i].captures))\"\"")
    end
```

Reply:
(Here's an example of how the behavior in the PR so far):
```julia
julia> res = ["a", "@b_str", "@c_str", "d", "e", "@f_str"];

julia> begin
           # to insert an entry like `raw""` for `"@raw_str"` in `res`
           ms = match.(r"^@(.*?)_str$", res)
           idxs = findall(!isnothing, ms)
           for i in idxs
               insert!(res, i, "$(only(ms[i].captures))\"\"")
           end
       end

julia> show(res)
["a", "b\"\"", "c\"\"", "@b_str", "@c_str", "f\"\"", "d", "e", "@f_str"]
```

and with `reverse`, they insert in the right spot:
```julia
julia> res = ["a", "@b_str", "@c_str", "d", "e", "@f_str"];

julia> begin
           # to insert an entry like `raw""` for `"@raw_str"` in `res`
           ms = match.(r"^@(.*?)_str$", res)
           idxs = findall(!isnothing, ms)
           for i in reverse(idxs)
               insert!(res, i, "$(only(ms[i].captures))\"\"")
           end
       end

julia> show(res)
["a", "b\"\"", "@b_str", "c\"\"", "@c_str", "d", "e", "f\"\"", "@f_str"]
```

Reply:
ah yes, good point, need to insert in the opposite side of the look up direction